force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -81,15 +53,7 @@ trigger AccountTrigger on Account (before insert, after insert) {
     * Email = 'default@email.com'
     * Trigger should only fire on insert.
     */    
-    if(Trigger.isAfter && Trigger.isInsert){     
-        List<Contact> contacts = new List<Contact>();   
-        for(Account acc : Trigger.new){
-            Contact con = new Contact();
-            con.LastName = 'DefaultContact';
-            con.Email = 'default@email.com';
-            con.AccountId = acc.Id;
-            contacts.add(con);
-        }
-        insert contacts; 
+    if(Trigger.isAfter && Trigger.isInsert){ 
+        AccountHelper.defaultContact(Trigger.new);
     }
 }
```

Comment:
Ideally here you are just making a call to some sort of trigger handler and no logic is in here at all. 

```
trigger Accounts on Account (before insert, before update, before delete,
        after insert, after update, after delete, after undelete) {
    new AccountTriggerHandler(Trigger.new).run();

}
```

Reply:
I'll take note of this. As I didn't implemented AccountTriggerHandler, resorted to call the methods directly here. But from next time, i'll follow this. Thanks.